spanner/api/Program.cs

```diff
@@ -0,0 +1,989 @@
+                // startTitle not provided, exclude startTitle and endTitle from 
+                // method call to use default values for both.
+                response = Task.Run(async () =>
+                    await QueryDataWithStoringIndexAsync(
+                        projectId, instanceId, databaseId));
+            }
+            else if (!string.IsNullOrEmpty(startTitle)
+                && string.IsNullOrEmpty(endTitle))
+            {
+                // Only startTitle provided, show message that user must 
+                // provide both startTitle and endTitle or exclude them 
+                // from their command.
+                Console.WriteLine("Please provide values for both the start "
+                    + "of the title index and the end of the title index.");
+                Console.WriteLine("Or you can exclude both to run the "
+                    + "query with default values.");
+                return 0;
+            }
+            else if (!string.IsNullOrEmpty(startTitle)
+                && !string.IsNullOrEmpty(endTitle))
+            {
+                // Both startTitle and endTitle provided, include them both in
+                // the method call to QueryDataWithIndexAsync.
+                response = Task.Run(async () =>
+                    await QueryDataWithStoringIndexAsync(
+                        projectId, instanceId, databaseId,
+                        startTitle, endTitle));
+            }
+            Console.WriteLine("Waiting for operation to complete...");
+            Task.WaitAll(response);
+            Console.WriteLine($"Operation status: {response.Status}");
+            return 0;
+        }
+
+        public static object AddColumn(string projectId,
+string instanceId, string databaseId)
+        {
+            var response = Task.Run(async () =>
+            await AddColumnAsync(
+                projectId, instanceId, databaseId));
+            Console.WriteLine("Waiting for operation to complete...");
+            Task.WaitAll(response);
+            Console.WriteLine($"Response status: {response.Status}");
+            return 0;
+        }
+
+        public static object WriteDataToNewColumn(string projectId,
+string instanceId, string databaseId)
+        {
+            var response = Task.Run(async () =>
+            await WriteDataToNewColumnAsync(
+                projectId, instanceId, databaseId));
+            Console.WriteLine("Waiting for operation to complete...");
+            Task.WaitAll(response);
+            Console.WriteLine($"Response status: {response.Status}");
+            return 0;
+        }
+
+        public static object QueryNewColumn(string projectId,
+string instanceId, string databaseId)
+        {
+            var response = Task.Run(async () =>
+            await QueryNewColumnAsync(
+                projectId, instanceId, databaseId));
+            Console.WriteLine("Waiting for operation to complete...");
+            Task.WaitAll(response);

```

Comment:
can't this just be:

QueryNewColumnAsync(projectId, instanceId, databaseId).Wait();

instead of task.run?